src/lib/Constants.ts

```diff
@@ -20,3 +20,6 @@ export const COLLATERAL_ASSET_ID = {
 export const ADDRESSES = {
   ZERO: '0x0000000000000000000000000000000000000000',
 };
+
+export const USDC_ADDRES_STAGING: string = '0x8707a5bf4c2842d46b31a405ba41b858c0f876c4';
```

Comment:
spelling nit: `USDC_ADDRESS_STAGING `

Reply:
Also instead of staging and production, can we use the terms ropsten and mainnet?

Reply:
Can you also note the difference between [actual ropsten USDC](https://goerli.etherscan.io/address/0x07865c6e87b9f70255377e024ace6630c1eaa37f) and this token (which is a test ERC20 we created)?